docs/install/remote.md

```diff
@@ -21,15 +21,33 @@ You can write your code on a local machine while running your functions on a rem
 
 Before you begin, ensure that the following prerequisites are met:
 
-1. Install MLRun locally.
+1. Applications:
+   - Supports pip and conda 
+   - Recommended pip 22.x+
+   - Python 3.8   
 
-    You need to install MLRun locally and make sure the that the MLRun version you install is the same as the MLRun service version. Install  a specific version using the following command; replace the `<version>`  placeholder with the MLRun version number (e.g., `1.0.0`):
+2. Install MLRun locally.
+
+    You need to install MLRun locally. Make sure the that the MLRun version you install is the same as the MLRun service version. Install a specific version using the following command; replace the `<version>`  placeholder with the MLRun version number (e.g., `1.0.0`):
  
     ```sh
     pip install mlrun==<version>
     ```
-
-    If you already installed a previous version of MLRun, upgrade it by running:
+	
+	There are a two `pip install` options:
+   - To install the requirements in the [requirements.txt](https://github.com/mlrun/mlrun/blob/development/requirements.txt), run:
+     `pip install mlrun` 
+   - If you expect to connect to, or work with, cloud providers (Azure/Google Cloud/S3), you can install additional packages. 
+      This is not part of the regular requirements since not all users work with those platforms. Using this option reduces the 
+      dependencies and the size of the installation. The additional packages include:
+     - S3 
+     - Azure Blob Storage
+     - Plotly
+      See the full list [here](https://github.com/mlrun/mlrun/blob/development/setup.py#L75). 
+      To install with the extras, run:
+     `pip install mlrun[complete]`
```

Comment:
```suggestion
   - If you expect to connect to, or work with, cloud providers (Azure/Google Cloud/S3), you can install additional packages. 
      This is not part of the regular requirements since not all users work with those platforms. Using this option reduces the 
      dependencies and the size of the installation. The additional packages include:
     -pip install mlrun[s3]       # Install requirements for  S3 
     -pip install mlrun[azure-blob-storage]     # install requirements for Azure blob storage
     -pip install mlrun[google-cloud-storage]      # install requirements for Google cloud storage
      See the full list [here](https://github.com/mlrun/mlrun/blob/development/setup.py#L75). 
      To install all extras, run:
     `pip install mlrun[complete]`
```